jooby/src/main/java/org/jooby/handlers/AssetHandler.java

```diff
@@ -185,11 +187,20 @@ public AssetHandler cdn(final String cdn) {
   }
 
   /**
-   * @param maxAge Set the cache header max-age value in seconds.
+   * @param maxAge Set the cache header max-age value.
    * @return This handler.
    */
-  public AssetHandler maxAge(final long maxAge) {
-    this.maxAge = maxAge;
+  public AssetHandler maxAge(final Optional<Duration> maxAgeOpt) {
```

Comment:
Yea, that is my point: it is disabled by default. So think we should allow to just enable, calling this method with an empty optional isn't useful.